test/acl_command_queue_test.cpp

```diff
@@ -674,15 +670,6 @@ MT_TEST(acl_command_queue, after_context_release) {
   // Should be able to release all the way.
   CHECK_EQUAL(CL_SUCCESS, clReleaseCommandQueue(cq0));
   CHECK_EQUAL(CL_SUCCESS, clReleaseCommandQueue(cq1));
-  CHECK_EQUAL(0, acl_ref_count(cq0));
-  CHECK_EQUAL(0, acl_ref_count(cq1));
-
-  ACL_LOCKED(CHECK(!acl_command_queue_is_valid(cq0)));
-  ACL_LOCKED(CHECK(!acl_command_queue_is_valid(cq1)));
-
-  // And once it's gone, it's gone.
-  CHECK_EQUAL(CL_INVALID_COMMAND_QUEUE, clReleaseCommandQueue(cq0));
-  CHECK_EQUAL(CL_INVALID_COMMAND_QUEUE, clReleaseCommandQueue(cq1));
```

Comment:
I have pushed a revised commit which includes the TSAN output for these lines.

Reply:
I am torn on this, but the commit log does get noisy with the entire TSAN output, and you already included most of the output in #71 anyway, so let's keep these longer stack traces in the referenced GitHub issues.